Code/CheXpertDataset.py

```diff
@@ -77,10 +79,3 @@ class UOnesTransform:
     def __call__(self, sample):
         sample = sample[label_columns].replace(-1, 1)
         return sample
-
-class GreyScale_to_RGB:
-    def __call__(self, im):
-        return tf.image.grayscale_to_rgb(
-            im,
-            name=None
-        )

```

Comment:
Weren't using this... ended up recreating it actually... but i'm using cv2 to match other preprocessing code.